1.2.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+
+// Task: ����������� ������ ����������>
+
+size_t n = 0;
+long* arr = NULL;
+
+void sortType()

```

Comment:
```suggestion
void eneteringArray(int* array, int size)
```
Странное название. Тип сортировки? Печать массива? (-1 балл)
Ну и аргументы должны быть, а не глобальная переменная.

Reply:
А если хочется в этой функции прочитать и размер массива, и элементы, то тип такая сигнатура будет
```
void eneteringArray(int** array, int* size)
```
Тогда в указатель `size` можно будет положить введенный размер, а в указатель `array` - массив.